src/test/java/nextstep/subway/acceptance/LineAcceptanceTest.java

```diff
@@ -0,0 +1,193 @@
+    @Test
+    void canFindSameNumberOfLinesWhenLinesWereCreated() {
+        // given
+        createLine(sinbundangLineCreationRequest);
+        createLine(bundangLineCreationRequest);
+
+        // when
+        var lineQueryResponse = RestAssured
+                .when()
+                    .get("/lines")
+                .then()
+                    .extract();
+        var lineNames = lineQueryResponse.jsonPath().getList("name");
+
+        // then
+        assertAll(
+                () -> assertThat(lineQueryResponse.statusCode()).isEqualTo(HttpStatus.OK.value()),
+                () -> assertThat(lineNames).containsExactlyInAnyOrder(
+                        sinbundangLineCreationRequest.getName(),
+                        bundangLineCreationRequest.getName()
+                )
+        );
+    }
+
+    /**
+     * Given 지하철 노선을 생성하고
+     * When 생성한 지하철 노선을 조회하면
+     * Then 생성한 지하철 노선의 정보를 응답받을 수 있다.
+     */
+    @DisplayName("지하철노선 조회")
+    @Test
+    void canGetResponseOfLineInformationByLineId() {
+        // given
+        var creationResponse = createLine(bundangLineCreationRequest);
+        var createdLineId = creationResponse.body().jsonPath().getLong("id");
+
+        // when
+        var lineQueryResponse = getLine(createdLineId);
+
+        // then
+        var id = lineQueryResponse.jsonPath().getLong("id");
+        var name = lineQueryResponse.jsonPath().getString("name");
+        var color = lineQueryResponse.jsonPath().getString("color");
+
+        assertAll(
+                () -> assertThat(lineQueryResponse.statusCode()).isEqualTo(HttpStatus.OK.value()),
+                () -> assertThat(id).isEqualTo(createdLineId),
+                () -> assertThat(name).isEqualTo(bundangLineCreationRequest.getName()),
+                () -> assertThat(color).isEqualTo(bundangLineCreationRequest.getColor())
+        );
+    }
+
+    /**
+     * Given 지하철 노선을 생성하고
+     * When 생성한 지하철 노선을 수정하면
+     * Then 해당 지하철 노선 정보는 수정된다
+     */
+    @DisplayName("지하철노선 수정")
+    @Test
+    void canModifyLineInformationWhichCreated() {
+        // given
+        var creationResponse = createLine(sinbundangLineCreationRequest);
+        var createdLineId = creationResponse.jsonPath().getLong("id");
+
+        // when
+        var modificationRequest = new LineModificationRequest("구분당선", "bg-red-600");
+        var modificationResponse = RestAssured
+                .given()
+                    .pathParam("lineId", createdLineId)
+                    .body(modificationRequest, ObjectMapperType.JACKSON_2)
+                    .contentType(MediaType.APPLICATION_JSON_VALUE)
+                .when()
+                    .put("/lines/{lineId}")
+                .then()
+                    .extract();
+
+        // then
+        var lineQueryResponse = getLine(createdLineId);
+        var modifiedName = lineQueryResponse.jsonPath().getString("name");
+        var modifiedColor = lineQueryResponse.jsonPath().getString("color");
```

Comment:
```suggestion
        var lineResponse = response.jsonPath().getObject(".", Line.class);
```
로 객체로 만들어서 검증할수도 있어요. 꺼내야 할 값이 하나~두개가 아니라 많아질 경우엔 고려해보세요!

Reply:
참고하겠습니다